Gemfile.lock

```diff
@@ -62,14 +62,14 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.0.4)
       loofah (~> 2.2, >= 2.2.2)
-    railties (5.2.1)
-      actionpack (= 5.2.1)
-      activesupport (= 5.2.1)
+    railties (5.2.2.1)
```

Comment:
### [railties](http://rubyonrails.org)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.2.1 → 5.2.2.1

[_[change-log](https://github.com/rails/rails/blob/v5.2.2.1/railties/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v5.2.2.1/railties)_]

:tada: Patched vulnerabilities:

- [CVE-2019-5420](https://nvd.nist.gov/vuln/detail/CVE-2019-5420)
  **Possible Remote Code Execution Exploit in Rails Development Mode**

  
  URL: https://groups.google.com/forum/#!topic/rubyonrails-security/IsQKvDqZdKw

  There is a possible a possible remote code executing exploit in Rails when in development mode. This vulnerability has been assigned the CVE identifier CVE-2019-5420.  Versions Affected:  6.0.0.X, 5.2.X. Not affected:       None. Fixed Versions:     6.0.0.beta3, 5.2.2.1  Impact ------ With some knowledge of a target application it is possible for an attacker to guess the automatically generated development mode secret token.  This secret token can be used in combination with other Rails internals to escalate to a remote code execution exploit.  All users running an affected release should either upgrade or use one of the workarounds immediately.  Releases -------- The 6.0.0.beta3 and 5.2.2.1 releases are available at the normal locations.  Workarounds ----------- This issue can be mitigated by specifying a secret key in development mode. In "config/environments/development.rb" add this:    config.secret_key_base = SecureRandom.hex(64)  Credits ------- Thanks to ooooooo_q






<details>
<summary>Commits</summary>

A change of **126** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v5.2.1...v5.2.2.1).

These are the first 10 commits:
- (cf90cba) [backport use_authenticated_cookie_encryption doc fixes [ci s…](https://github.com/rails/rails/commit/cf90cba2becaa971b1b9140bbe83bc4b101d8187)
- (10679c2) [Merge pull request #33325 from Edouard-chin/ec-deprecate-cla…](https://github.com/rails/rails/commit/10679c2cbd25550f6625dcac0d9f813e8e91b2ab)
- (dd11336) [Avoid extra scoping when using `Relation#update`](https://github.com/rails/rails/commit/dd1133699f7266e9659c3624de1c77fcd43a4691)
- (ede8bc0) [Merge pull request #33489 from Atul9/remote-ip](https://github.com/rails/rails/commit/ede8bc0c77016bb93b8a2957fdb941c3b2bb0e59)
- (9897cf5) [Merge pull request #33488 from znz/avoid-to-post-twice](https://github.com/rails/rails/commit/9897cf55e2609e1e67ec0cb50ea7f2cff95fcc1d)
- (0d2cc93) [Merge pull request #33452 from thetizzo/rails_32_to_40_doc_u…](https://github.com/rails/rails/commit/0d2cc930acd1ebe5f0c1cb7729088ebbde2c2452)
- (067913f) [Merge pull request #33507 from jackc/patch-1](https://github.com/rails/rails/commit/067913f95d38f6acd11223f6b35f9386c31c94ca)
- (ccf90fb) [Fix uploading Tempfiles to Azure Storage](https://github.com/rails/rails/commit/ccf90fb2e0a4717e63bad87e87409facc3d3b2da)
- (250fc41) [Tempfile.open requires a name in Ruby 2.3](https://github.com/rails/rails/commit/250fc41df47a44545a68f82429f21c118a224a21)
- (c32462a) [Merge branch '5-2-1' into 5-2-stable](https://github.com/rails/rails/commit/c32462a0db5005961f2f83973e44d90c6ad86a5f)


</details>